### DISCUSSION TITLE
MarkerIcon tap event listener

### DISCUSSION THREAD

**Discussion by jesussmile** [liodali/osm_flutter#378] [Ideas]
Hi! Liodali, are you trying to update a marker Icon click listener ?  For example when a user will click on a marker there will be a pop up giving more information about the marker ? I am getting some mixed results. I just wanted to know if you are working on it ? if not i will also try on my own.

I dont see much being done on `onMarkerCick`

**Comment by liodali** [comment]
the only issue that we have is that in ios we cannot show popup when click om marker
that why i put that feature in hold, in general im trying to make popup built with flutter 
not natively like we have now for example in web or osmdroid we can show native popup but i don't like that way because we will be limited only to show text

**Reply by jesussmile** [reply]
I think we can use pixel coordinates instead of geo point coordinates, All we need is to find the pixel value of the coordinates then put the markerIcon from flutter side on those pixel coordinates. This way it can be done for all the platform. I am trying to find where i can add that in your code, do you have any idea ?

**Reply by jesussmile** [reply]
is there anyway i can get pixel coordinates when I click on the map ?

**Reply by liodali** [reply]
don't forget that map render natively if we put the marker in flutter view we need to move it also the marker when the map moves

**Reply by liodali** [reply]
there are some apis to do that but those apis are not expose it in our plugin since we don't need that

**Comment by jesussmile** [comment]
Okay got you!. Hopefully we will find a solution soon. Thanx
